docs/book/src/reference/providers.md

```diff
@@ -26,6 +26,7 @@ updated info about which API version they are supporting.
 - [Tencent Cloud](https://github.com/TencentCloud/cluster-api-provider-tencent)
 - [vSphere](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere)
 - [Alibaba Cloud](https://github.com/oam-oss/cluster-api-provider-alicloud)
+- [Azure Stack HCI](https://github.com/microsoft/cluster-api-provider-azurestackhci)
```

Comment:
not sure why Alibaba Cloud is at the bottom but the rest is alphabetically sorted so let's move this up to after Azure

Reply:
I assume it is OK if I move Alibaba cloud as well then so that it becomes consistent?